zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/remote/RemoteInterpreterServer.java

```diff
@@ -482,9 +490,14 @@ public void close(String sessionId, String className) throws InterpreterRPCExcep
           Iterator<Interpreter> it = interpreters.iterator();
           while (it.hasNext()) {
             Interpreter inp = it.next();
-            if (inp.getClassName().equals(className)) {
+            LazyOpenInterpreter lazy = (LazyOpenInterpreter) inp;
```

Comment:
Is this cast safe? I would prefer an Instanceof test before making this cast.

Reply:
Thanks for your suggestion, Reamer, I will add the condition.